public/jira/releases/semver-releases.ts

```diff
@@ -0,0 +1,74 @@
+import semver from "semver";
+import uniqueTrailingNames from "./unique-trailing-names";
+
+interface UnsortedRelease {
+  release: string;
+}
+
+interface SemverRelease {
+  release: string;
+  shortName: string;
+  version: string | undefined;
+  shortVersion: string | undefined;
+}
+
+function partialReleaseName(release: string) {
```

Comment:
just flagging this pr too - we need to add return types if that's the direction we go